_assignments/mp4.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: assignment
+title: "MP4: Flight Simulator"
+parts:
+  -
+    title: MP4
+    due: March 25, 2019 @ 11:59 PM
+material:
+  -
+    name: Starting Unity package
+    url: https://courses.engr.illinois.edu/cs498vr3/fa2018/mps/MP4v2.unitypackage
+rubric:
+  -
+    name: Plane
+    points: 5
+    description: Game has a controllable plane
+  -
+    name: Acceleration/Deceleration
+    points: 5
+    description: Plane can accelerate and decelerate comfortably
+  -
+    name: Rotation
+    points: 10
+    description: Plane can yaw, pitch, and roll
+  -
+    name: Uses Touch Controllers
+    points: 25
+    description: Plane control scheme makes inventive use of rotation and position tracking of touch controllers
+  -
+    name: Weapon
+    points: 10
+    description: Pressing a button shoots out a raycast or bullet from the plane
+  -
+    name: Spheres
+    points: 5
+    description: There are spheres around the terrain
+  -
+    name: Shooting Game
+    points: 25
+    description: Player can shoot spheres, which disappear, then reappear
+  -
+    name: Tutorial
+    points: 10
+    description: Game has an in-world tutorial that clearly communicates how to control the plane
+  -
+    name: Start Menu
+    points: 10
+    description: Game has a legible and understandable in-world start menu
+  -
+    name: Laser pointer
+    points: 20
+    description: The Start Menu is controlled by an in-game, grabbable laser pointer
+  -
+    name: Displaying Score
+    points: 5
+    description: Score is displayed in-world
+  -
+    name: Sound
+    points: 20
+    description: Elements of the simulator that should have sound, do.
+  -
+    name: Game over
+    points: 10
+    description: Game handles plane crashes in some manner
+  -
+    name: Framerate/Comfort
+    points: 40
+    description: Framerate rarely drops below 60 fps, game sound is not obnoxious, and plane acceleration, deceleration, and rotations are comfortable. Game conforms to Oculus Best Practices Guide
+  -
+    name: Interactive tutorial
+    points: 20
+    extra: true
+    description: The tutorial provided to the player is interactive (Extra Credit)
+
+points: 200
+---
+## Overview
+Welcome to MP4! In this MP, you are going to build a flight simulator in Unity. You are going to construct a start menu and tutorial, write script to support your plane, make a UI, and optimize the game to fulfill requirements from the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/)! 

```

Comment:
recommend to add line break under this header to be consistent with all the other ones